src/platform/plugins/shared/discover/public/application/main/components/single_tab_view/single_tab_view.tsx

```diff
@@ -127,8 +126,8 @@ export const SingleTabView = ({
       : { loading: true }
   );
 
-  useMount(() => {
-    if (!currentStateContainer || !currentCustomizationService) {
+  useEffect(() => {
```

Comment:
Switching to an effect means we'll reinitialize a tab if its runtime state is reset, although that shouldn't really happen now since we regenerate IDs when copying, which already gives us a fresh runtime state.